notifications/rest/users/create-user/create-user.7.x.java

```diff
@@ -0,0 +1,27 @@
+// NOTE: This example uses the next generation Twilio helper library - for more
+// information on how to download and install this version, visit
+// https://www.twilio.com/docs/libraries/java
+import com.twilio.Twilio;
+import com.twilio.rest.notify.service.Notification;
```

Comment:
This import should be `NotifyUser`. Also you are missing an import for the ArrayList:
`import java.util.ArrayList;`